src/ui/hooks/FlowRouter.tsx

```diff
@@ -0,0 +1,205 @@
+import React, {useEffect, useRef, useReducer, Suspense} from 'react';
+import FlowDispatchTypes from '../enums/FlowDispatchTypes';
+import {AuthOption} from '../interfaces/AuthOptionInterfaces';
+import FlowDispatchContext from '../contexts/FlowDispatchContext';
+import {Flow} from '../interfaces/FlowSelectorInterfaces';
+import {CONSTANTS} from '../../constants/constants';
+
+import {ComponentStoreGet, ComponentStoreSet, ComponentStoreMethods, FlowAction, ComponentStore, ComponentMap} from '../interfaces/FlowRouterInterfaces';
+
+const options: AuthOption[] = CONSTANTS.verification_options;
+const useMenu: boolean = options.length > 1;
+
+// let flow: Flow;
+
+const FlowController: React.FC<{}> = () => {
+    const flow = useRef<Flow>(createFlow(0));
+    const componentStore = useRef<ComponentStore>(initComponentStore());
+
+    const [state, dispatch] = useReducer(flowReducer, {
+        step: 'confirmation',
+        authIndex: 0
+    });
+
+    const prevStep: string = flow.current[state.step]![FlowDispatchTypes.BACK] ?? '';
+
+    let TheComponent = renderScreen(state.step),
+        componentStoreMethods: ComponentStoreMethods = createComponentStoreConnection(state.step);
```

Comment:
Not sure I understand the concern - they're on two different lines, and there's nothing stopping someone from adding variables within this `let` declaration. Can you clarify?

Reply:
If you have to move any of the lines along the file you will get more lines changed since line 26 no longer ends in `;`. An opposite situation from enabling `,` after the last element of each array so that new items don't trigger two line changes.